requirements/dev.txt

```diff
@@ -0,0 +1,9 @@
+coverage==3.7.1
+django-debug-toolbar==1.4
+django-devserver==0.8.0
+gunicorn==19.4.1
```

Comment:
Yeah we recently started to use it in development to be able to use whitenoise and have a dev/prod parity for serving static files.